qdrant-landing/content/articles/bloop.md

```diff
@@ -4,6 +4,9 @@ short_description: bloop is a fast code-search engine that combines semantic sea
 description: bloop is a fast code-search engine that combines semantic search, regex search and precise code navigation
 social_preview_image: /case_studies_data/bloop/social_preview.png
 preview_dir: /case_studies_data/bloop/preview
```

Comment:
It might be better to move the images to `articles_data` as well if we don't want to have case studies separated

Reply:
will do